<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>699</width>
    <height>551</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="SPINPRZYCISK">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>80</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>SPINEK</string>
    </property>
   </widget>
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>421</width>
      <height>261</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="maszyna"/>
   </widget>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>10</y>
      <width>241</width>
      <height>171</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QWidget" name="widget" native="true">
       <widget class="QLineEdit" name="JapkoEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="BananEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="WinogronkoEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>60</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="WisniaEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>80</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="AnanasEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>100</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="KiwiEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>120</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="BonusEdit">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>140</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekJapko">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>üçé</string>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekBanan">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>40</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>üçå</string>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekWinogrono">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>60</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>üçá</string>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekWisnia">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>80</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>üçí</string>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekAnanas">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>üçç</string>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekKiwi">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>120</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>ü•ù</string>
        </property>
       </widget>
       <widget class="QLabel" name="ObrazekBonus">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>16</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>üéÅ</string>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>0</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Szanse</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Kiwi">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>120</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Wisnia">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>80</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Bonus">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>140</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Banan">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Ananas">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>100</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Japko">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="liczbaSymboliLineEdit_Winogronko">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>60</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Japko">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>20</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Wisnia">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>80</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Ananas">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>100</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Banan">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>40</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Winogronko">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>60</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Bonus">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>140</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="kwotaWygranejLineEdit_Kiwi">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>120</y>
          <width>61</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>100</x>
          <y>0</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Ilosc</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>170</x>
          <y>0</y>
          <width>41</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>Mnoznik</string>
        </property>
       </widget>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="plusButton">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>370</y>
      <width>21</width>
      <height>18</height>
     </rect>
    </property>
    <property name="text">
     <string>+</string>
    </property>
   </widget>
   <widget class="QPushButton" name="minusButton">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>370</y>
      <width>21</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>-</string>
    </property>
   </widget>
   <widget class="QLabel" name="StawkaLabel">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>370</y>
      <width>37</width>
      <height>12</height>
     </rect>
    </property>
    <property name="layoutDirection">
     <enum>Qt::LayoutDirection::LeftToRight</enum>
    </property>
    <property name="text">
     <string>SpinPrice</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignHCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="saldoLabel">
    <property name="geometry">
     <rect>
      <x>470</x>
      <y>250</y>
      <width>71</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Kasa</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="infoLabel">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>340</y>
      <width>101</width>
      <height>61</height>
     </rect>
    </property>
    <property name="text">
     <string>Wygrane</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignmentFlag::AlignCenter</set>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>699</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
